drivers/main-device.js

```diff
@@ -316,7 +313,6 @@ module.exports = class mainDevice extends Homey.Device {
             const myTime = timeNow.toLocaleString('en-US', { hour: '2-digit', minute: '2-digit', hour12: false, timeZone: myTZ });
             const myDate = timeNow.toLocaleString('en-US', { day: '2-digit', month: '2-digit', year: 'numeric', timeZone: myTZ });
             const myTimeMinutes = Number(myTime.split(':')[0]) * 60 + Number(myTime.split(':')[1]);
-            const spaTimeMinutes = (hour * 60) + minute;
```

Comment:
Is this an accidental removal? spaTimeMinutes is used in the next confition.

Reply:
not intended. Did some formatting with prettier and reverted that later. Missed this bit